src/graphql/mutations/__tests__/__snapshots__/UpdateUser.js.snap

```diff
@@ -1,5 +1,155 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`UpdateUser cannot set duplicated slug 1`] = `
+Array [
+  [GraphQLError: Slug already taken],
+]
+`;
+
+exports[`UpdateUser cannot set duplicated slug 2`] = `
+Object {
+  "email": "user1@example.com",
+  "facebookId": "fbid123",
+  "githubId": "githubId123",
+  "id": "testUser1",
+  "name": "test user 1",
+  "slug": "test-user-1",
+  "updatedAt": "2017-01-28T08:46:07.011Z",
```

Comment:
If `UpdateUser` does not update the user, why would `updatedAt` exist here?

Reply:
`updatedAt` is returned by `GetUser`
I'm checking that `updatedAt` shouldn't be changed since last test file, and that's why MockDate is different for all test cases (line 14 at __tests__/UpdateUser.js

but let me change current time to a more readable time